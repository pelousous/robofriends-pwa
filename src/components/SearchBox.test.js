import React from 'react';
import {shallow} from 'enzyme';
import SearchBox from './SearchBox';

it('expect SearchBox component to be rendered', () => {
    const wrapper = shallow(<SearchBox/>);
    const input = wrapper.find('input');
    expect(wrapper).toMatchSnapshot();
    expect(input.length).toEqual(1);
    //input.simulate('change',{ target: { value: 'Changed' } });
    //expect(input.props().value).toEqual('Changed');
})