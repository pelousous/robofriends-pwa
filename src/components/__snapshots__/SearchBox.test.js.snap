// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect SearchBox component to be rendered 1`] = `
<div
  className="pa2"
>
  <input
    aria-label="Search"
    className="pa3 ba b--green bg-lightest-blue"
    placeholder="search robots"
    type="search"
  />
</div>
`;
